Added tools - Refund analytics and filter orders by amount

santhoshJus opened the pull request:
Added tools for refund analytics and to filter orders by amount.

Fetching details from kvrefundtxns for refund analytics similar to how it was implemented in lighthouse

murdore commented:
@santhoshJus this needs to be added in ligthouse. we will be moving to consuming tools from lighthouse via sse

santhoshJus commented:
Added tool in lighthouse: [BZ-44883: feat: Added tool to filter orders by amount](https://bitbucket.juspay.net/projects/BZ/repos/lighthouse/pull-requests/3287/overview)

santhoshJus committed:
Added tools - Refund analytics and filter orders by amount
